src/main/java/az/edu/turing/modul1/Task3C.java

```diff
@@ -0,0 +1,76 @@
+package az.edu.turing.modul1;
+
+import java.util.Arrays;
+import java.util.Random;
+import java.util.Scanner;
+
+public class Task3C {
+    public static void main(String[] args) {
+
+
+        Scanner sc = new Scanner(System.in);
+        String[][] board = new String[5][5];
+        Random random = new Random();
+
+        int targetRow = random.nextInt(5);
+        int targetColumn = random.nextInt(5);
+
+        System.out.println("All set. Get ready to rumble!");
+        int count = 0;
+
+        for (int i = 0; i < board.length; i++) {
+            Arrays.fill(board[i], " - ");
+
+        }
+
+        while (true) {
+
+            for (int i = 0; i < board.length; i++) {
+                for (int j = 0; j < board[i].length; j++) {
+                    System.out.print(board[i][j]);
+                }
+                System.out.println();
+            }
+
+            System.out.print("Enter the row (1-5): ");
+            int row = sc.nextInt() - 1;
+            while (row < 0 || row > 4) {
```

Comment:
you should move this validation logic to another method

Reply:
I fixed